galley/pkg/source/kube/builtin/types.go

```diff
@@ -157,6 +157,32 @@ var (
 				return out, nil
 			},
 		},
+		"Namespace": {
+			spec:    getSpec("Namespace"),
+			isEqual: resourceVersionsMatch,
+			extractObject: func(o interface{}) metav1.Object {
+				if obj, ok := o.(metav1.Object); ok {
+					return obj
+				}
+				return nil
+			},
+			extractResource: func(o interface{}) proto.Message {
+				if obj, ok := o.(*v1.Namespace); ok {
+					return obj
```

Comment:
This needs to be &obj.Spec. that should take care of the panic.

Reply:
Thanks! that fixed it.